po/es.po

```diff
@@ -449,12 +455,12 @@ msgid ""
 "  Library prefix:    %s\n"
 "  Compiled for:      %s\n"
 msgstr ""
-"Opciones de compilado:\n"
-"  \"Release\" versión:      %s (%s)\n"
-"  BinReloc habilitado:    %s\n"
-"  Prefijo de datos:       %s\n"
-"  Prefijo de bibliotecas: %s\n"
-"  Compilado en:           %s\n"
+"Opciones de compilación:\n"
+"  Versión de producción: %s (%s)\n"
+"  BinReloc habilitado:   %s\n"
+"  Prefijo de datos:      %s\n"
+"  Prefijo de biblioteca: %s\n"
```

Comment:
this are a path so means plural here!

Reply:
Do you mean "Prefijos de datos" and "Prefijos de bibliotecas"?

Reply:
There are path to libraries! not Library! so "Prefixo de librerias" due all the LIBRARIES (not book library) are under that path. so then, it's a context problem! also "Library" not "biblioteca" as Borlandc try to make sense! this are a standar for C/C++

Reply:
I know what paths and libraries are. I see no context problem here AFAIK. Borlanc is a library and the files under that path are also libraries because a library is a set of constants, variables, and functions to be used by programs or other libraries. So for me "Prefijo de bibliotecas" or even better "Prefijo de las bibliotecas", meaning the path the libraries are under, is the right translation. You're right in that the number was wrong, it has to be a plural form.